src/components/v2/V2Project/index.tsx

```diff
@@ -75,51 +75,62 @@ export default function V2Project({
     isArchived,
     projectOwnerAddress,
     handle,
+    loading,
   } = useContext(V2ProjectContext)
   const {
     currencies: { ETH },
   } = useContext(CurrencyContext)
 
-  const isMobile = useMobile()
-
-  const canReconfigureFundingCycles = useV2ConnectedWalletHasPermission(
-    V2OperatorPermission.RECONFIGURE,
-  )
+  // Checks URL to see if user was just directed from project deploy
+  const router = useRouter()
+  const isNewDeploy = Boolean(router.query.newDeploy)
 
+  const [newDeployModalVisible, setNewDeployModalVisible] =
+    useState<boolean>(isNewDeploy)
+  const [balancesModalVisible, setBalancesModalVisible] =
+    useState<boolean>(false)
   const [handleModalVisible, setHandleModalVisible] = useState<boolean>()
   const [payAmount, setPayAmount] = useState<string>('0')
   const [payInCurrency, setPayInCurrency] = useState<CurrencyOption>(ETH)
 
-  const { data: queuedFundingCycleResponse } = useProjectQueuedFundingCycle({
+  const isMobile = useMobile()
+  const canReconfigureFundingCycles = useV2ConnectedWalletHasPermission(
+    V2OperatorPermission.RECONFIGURE,
+  )
+  const {
+    data: queuedFundingCycleResponse,
+    loading: queuedFundingCycleLoading,
+  } = useProjectQueuedFundingCycle({
     projectId,
   })
-
   const [queuedFundingCycle] = queuedFundingCycleResponse || []
 
-  const editV2ProjectDetailsTx = useEditV2ProjectDetailsTx()
-
-  // Checks URL to see if user was just directed from project deploy
-  const router = useRouter()
-  const isNewDeploy = Boolean(router.query.newDeploy)
-
   const converter = useCurrencyConverter()
-
+  const editV2ProjectDetailsTx = useEditV2ProjectDetailsTx()
   const hasEditPermission = useV2ConnectedWalletHasPermission(
     V2OperatorPermission.RECONFIGURE,
   )
-
   const isOwner = useIsUserAddress(projectOwnerAddress)
 
-  const [newDeployModalVisible, setNewDeployModalVisible] =
-    useState<boolean>(isNewDeploy)
-  const [balancesModalVisible, setBalancesModalVisible] =
-    useState<boolean>(false)
-
   const colSizeMd = singleColumnLayout ? 24 : 12
+
+  const allFundingCyclesLoading =
+    loading.fundingCycleLoading || queuedFundingCycleLoading
   const hasCurrentFundingCycle = fundingCycle?.number.gt(0)
   const hasQueuedFundingCycle = queuedFundingCycle?.number.gt(0)
+  const showRelaunchFundingCycleBanner =
+    !allFundingCyclesLoading &&
```

Comment:
this file diff is just reorganisation, **except for** this line. This line is the only meaningful change here.